build.gradle

```diff
@@ -17,6 +18,9 @@ apply from: file('upload.gradle')
 subprojects {
     if(!project.name.equals('jme3-android-examples')) {
         apply from: rootProject.file('common.gradle')
+        if (!['jme3-testdata', 'sdk'].contains(project.name)) {
```

Comment:
we could upload them, so there are available on the org.jmonkeyengine repository. But choose to not "link" them to jcenter. linking is a per artifact action, that should be done on bintray.